jobs/bits-service/templates/statsd_monitor

```diff
@@ -7,7 +7,7 @@ require 'socket'
 socket = UDPSocket.new
 socket.bind('127.0.0.1', 8125)
 
-while true
```

Comment:
I think we accidentially picked this up when creating the WIP branch. The change itself is fine, but could you please split this off into a new PR?